packages/office-ui-fabric-react/src/components/Dialog/DialogContent.tsx

```diff
@@ -35,10 +35,11 @@ export class DialogContent extends BaseComponent<IDialogContentProps, IDialogCon
   public render() {
     let {
       showCloseButton,
-      closeButtonAriaLabel,
+      closeButtonAriaLabel = 'Close',
```

Comment:
I'm fine leaving this out if we want to keep it 100% language neutral, but user are obviously able to set it to their own string.

So I guess the question is what is worse, no label, or label in non-user language.